### DISCUSSION TITLE
Malformed input or input contains unmappable characters - utf-8

### DISCUSSION THREAD

**Discussion by dshershov** [apache/ozone#4984] [FAQ] [answered]
Hi everyone!
I stuck with the next issue:
`java.nio.file.InvalidPathException: Malformed input or input contains unmappable characters: /s3v/buckettest/ba?ket1` - where `bA?ket` have a utf-8 symbol
And now, ozone manager don't starting due to issue
How can we get around this syntax error?

**Comment by dshershov** [comment, marked as answer]
looks like workaround for me
![waound 2023-06-27 в 15 23 00](https://github.com/apache/ozone/assets/17449010/61c7b0a2-3c04-4647-a88f-7fa3413b9562)

**Reply by adoroszlai** [reply]
Thanks @dshershov for posting the solution here.  It seems you have hit HDDS-5455.
